Sources/PCRoomDelegate.swift

```diff
@@ -3,6 +3,7 @@ import PusherPlatform
 
 public protocol PCRoomDelegate: AnyObject {
     func onMessage(_ message: PCMessage)
+    func onMultipartMessage(_ message: PCMultipartMessage)
```

Comment:
I'm not sure how I feel about `onMultipartMessage`, but I can see how the whole room delegate thing might force our hand on this one. Is @mdpye doing this in kotlin? It might be too weird if Swift is the only SDK with the new hook name. Can we avoid it? (I guess you might need two room delegate types, which is a bit icky... 🤔

Reply:
I tried implementing this like I discussed with @mdpye but because of the structure of the SDK it is definitely icky and it proved to be a much more deep rooted change than we initially thought :(